Buildings/Experimental/NatVentControl/Lockouts/DryBulbLockout.mo

```diff
@@ -0,0 +1,91 @@
+within Buildings.Experimental.NatVentControl.Lockouts;
+block DryBulbLockout
+  "Locks out natural ventilation if outdoor air dry bulb temperature is below a user-specified threshhold"
+  parameter Real TOADB(min=0,
+    final unit="K",
+    final displayUnit="K",
+    final quantity="Temperature")=288.7 "Outdoor air temperature below which nat vent is not allowed";
+
+  Controls.OBC.CDL.Continuous.Hysteresis hys(
+    uLow=0,
+    uHigh=0.1,
+    pre_y_start=true)
+    "Tests if VAV setpoint minus wet bulb temp is greater than user-specified tolerance (typically 8 F). If so, window can open. If not, window must be closed."
+    annotation (Placement(transformation(extent={{-20,20},{0,40}})));
+  Controls.OBC.CDL.Continuous.Add add2
+    annotation (Placement(transformation(extent={{-60,20},{-40,40}})));
+  Controls.OBC.CDL.Continuous.ChangeSign           chaSig
+    "Negates room temperature setpoint"
+    annotation (Placement(transformation(extent={{-90,60},{-70,80}})));
+  Controls.OBC.CDL.Logical.And and2
+    annotation (Placement(transformation(extent={{60,20},{80,40}})));
+  Controls.OBC.CDL.Continuous.Hysteresis hys1(
+    uLow=TOADBLo,
+    uHigh=TOADB,
+    pre_y_start=false)
+    annotation (Placement(transformation(extent={{0,-60},{20,-40}})));
+  Controls.OBC.CDL.Logical.Not not1
+    annotation (Placement(transformation(extent={{20,20},{40,40}})));
+  Controls.OBC.CDL.Interfaces.RealInput TDryBul
+    "Outdoor air dry bulb temperature"
+    annotation (Placement(transformation(extent={{-140,-30},{-100,10}}),
+        iconTransformation(extent={{-140,20},{-100,60}})));
+  Controls.OBC.CDL.Interfaces.BooleanOutput natVenSigOAT
```

Comment:
use `yLocOut` as the name of all the lock out outputs.

Reply:
@JayHuLBL , Does it work if we add additional characters for each lockout, e.g. "yDryBulLoc" for the dry bulb temp lockout? I was hoping to be able to distinguish between lockout signals for unit tests, etc, where our test plot is the different lockouts plotted together over time. Ideally, I also wanted to indicate that these lockouts were for natural ventilation (to distinguish them from radiant lockouts), e.g. "yDryBulNatVenLoc". 
If this isn't possible, I can find a workaround, but I wanted to check.

Reply:
Also, my initial reasoning for using "sig" rather than "loc" is that a true signal indicates that natural ventilation is allowed (not that it is locked out. Is it still best to use "loc"?